src/applications/personalization/profile/tests/e2e/helpers.js

```diff
@@ -94,3 +94,15 @@ export function nameTagRendersWithoutDisabilityRating() {
   cy.findByText(/View disability rating/i).should('not.exist');
   cy.findByText(/service connected/i).should('not.exist');
 }
+
+export function registerCypressHelpers() {
+  // The main loading indicator is shown and is then removed
+  Cypress.Commands.add('loadingIndicatorWorks', () => {
```

Comment:
Now that this helper exists we can use it in a lot of existing Profile tests that check for a loading spinner to be visible and then disappear.

I will likely add a lot more new Cypress helpers here so that we can use them throughout the Profile tests.

Reply:
excellent. If you need help on any helpers let me know. I don't have much knowledge yet of what needs these, but I'm sure I could help if needed